tasks/elasticsearch-RedHat-version-lock.yml

```diff
@@ -2,6 +2,19 @@
 - name: RedHat - install yum-version-lock
   become: yes
   yum: name=yum-plugin-versionlock state=present update_cache=yes
+
 - name: RedHat - lock elasticsearch version
   become: yes
   shell: yum versionlock delete 0:elasticsearch* ; yum versionlock add {{ es_package_name }}{% if es_version is defined and es_version != "" %}-{{ es_version }}{% endif %}
+  when: es_version_lock
+
+- name: RedHat - check elasticsearch version lock
+  become: yes
+  shell: yum versionlock list | grep -c elasticsearch
```

Comment:
`grep -c elasticsearch` is failing if it doesn't find elasticsearch package.

Reply:
I've forgot to add `failed_when: False` and `changed_when: False` to this task. Should be fine now.